static/images/svg-icons/network.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M462.862 103a49.138 49.138 0 0 0-48.049 39.761H199.942A49.138 49.138 0 0 0 151.893 103c-26.513 0-48.61 22.1-48.61 49.714a48.661 48.661 0 0 0 18.785 38.657l-60.754 120.4a46.064 46.064 0 0 0-12.144-1.664A49.418 49.418 0 0 0 0 359.843 48.912 48.912 0 0 0 49.154 409c23.746 0 43.633-17.124 48.046-40.324h214.314c4.416 23.2 24.3 40.321 48.049 40.321a49.461 49.461 0 0 0 30.929-87.811l61.314-120.4a67.162 67.162 0 0 0 11.04 1.1A49.269 49.269 0 0 0 512 152.733C512 125.676 489.919 103 462.862 103zM96.659 349.347a49.066 49.066 0 0 0-17.681-28.721l60.21-120.4a48.38 48.38 0 0 0 13.249 1.664 46.159 46.159 0 0 0 27.057-8.288l139.748 138.64a39.13 39.13 0 0 0-7.184 17.121c.016-.016-215.399-.016-215.399-.016zm276.169-37.553a48.332 48.332 0 0 0-13.249-1.664 50.4 50.4 0 0 0-27.057 8.288L193.334 179.774a48.757 48.757 0 0 0 6.624-17.681h214.855c2.768 12.7 9.392 23.2 19.329 30.385z"/>
```

Comment:
Not sure what happened to this one, but the path's current definition has it sitting totally outside of the `viewBox`,:

<img width="128" alt="Screen Shot 2019-12-04 at 10 45 27 PM" src="https://user-images.githubusercontent.com/1048442/70202493-da109e00-16e7-11ea-9870-5ebe450af418.png">

(The `viewBox` is the white square when I opened the SVG in Illustrator.)

That's why the icon isn't visible at all on your staged branch. It should work fine if you use this path instead:

```suggestion
<path d="M16.272,3.621a1.729,1.729,0,0,0-1.689,1.4H7.029a1.714,1.714,0,0,0-3.4.35,1.71,1.71,0,0,0,.66,1.359L2.156,10.961a1.623,1.623,0,0,0-.427-.059A1.738,1.738,0,0,0,0,12.648v0a1.72,1.72,0,0,0,1.711,1.728h.017a1.725,1.725,0,0,0,1.689-1.418h7.535a1.724,1.724,0,1,0,2.776-1.669l2.156-4.233a2.526,2.526,0,0,0,.388.039,1.739,1.739,0,0,0,0-3.477ZM3.4,12.282a1.723,1.723,0,0,0-.621-1.01L4.893,7.039a1.707,1.707,0,0,0,.466.059,1.615,1.615,0,0,0,.951-.292l4.913,4.874a1.373,1.373,0,0,0-.252.6Zm9.709-1.321a1.741,1.741,0,0,0-.466-.058,1.77,1.77,0,0,0-.951.291L6.8,6.32A1.732,1.732,0,0,0,7.03,5.7h7.553a1.749,1.749,0,0,0,.68,1.068Z" fill="url(#paint0_linear8)"/>
```